src/engine/strat_engine/pool.rs

```diff
@@ -325,51 +289,48 @@ impl StratPool {
         };
 
         match result {
-            ThinPoolStatus::Good(wstatus, usage) => {
-                match wstatus {
-                    ThinPoolWorkingStatus::Good => {}
-                    ThinPoolWorkingStatus::ReadOnly => {
-                        // TODO: why is pool r/o and how do we get it
-                        // rw again?
-                    }
-                    ThinPoolWorkingStatus::OutOfSpace => {
-                        // TODO: Add more space if possible, or
-                        // prevent further usage
-                        // Should never happen -- we should be extending first!
+            ThinPoolStatus::Good => {}
```

Comment:
This is related to the problem of ```thin_pool_status()``` behaviour. It is hard to comment, since I think that the type being matched on is wrong.